### PR TITLE
Small UI fixes

### PR DESCRIPTION
**Pull request by jacogr**
- Actually retrieve saved dapp visibility from storage (previously refreshing the page lost the visible apps, storage never taken into account)
- Align SVG address icons on MethodDecoding (previous these were displayed too low for null & contract addresses - also Closes https://github.com/ethcore/parity/issues/3954)
- Improve isNullAddress check (actual full value)

### CODE REVIEW

**Comment by ngotchac** [comment]
Using `null` for every address in InputAddress is misleading. First, it breaks adding a new address to the address book. It also will result in a wrong information for transaction to the Zero Account for example. Shouldn't it be limited to informations in contracts ?

**Comment by jacogr** [comment]
Made isNullAddress explicit & more robust

**Comment by ngotchac** [comment]
The only grumble is that in the Add Address Modal, when you enter `0x000000000000000000000000000000000000000` then another `0`, the input is changed to `null`, which is a bit misleading : you cannot add the Zero Account to your address book.

**Comment by jacogr** [comment]
Can add it, it just displays 'null' as the name. Made it so it doesn't display the name on input, only when queried.

File: js/src/ui/Form/InputAddress/inputAddress.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ class InputAddress extends Component {
     classes.push(!icon ? styles.inputEmpty : styles.input);
 
     const containerClasses = [ styles.container ];
-    const nullName = value && new BigNumber(value).eq(0) ? 'null' : null;
+    const nullName = readOnly && isNullAddress(value) ? 'null' : null;
```
Comment (ngotchac) [inline js/src/ui/Form/InputAddress/inputAddress.js]: Might want to check `disabled` as well (it's the one used in contracts)

**Pull request merged**